src/routes/github/create-branch/github-remove-session.ts

```diff
@@ -0,0 +1,15 @@
+import { Request, Response } from "express";
+
+
+export const GithubRemoveSession = (req: Request, res: Response) => {
+	const { gitHubAppConfig } = res.locals;
+
+	if (!gitHubAppConfig) {
+		res.sendStatus(401);
+		return;
+	}
+
+	req.session.githubToken = undefined;
```

Comment:
If i refresh the page after this it recreates my session, you might have to use something like `req.session.destroy`